src/Init/Data/Int/DivMod/Lemmas.lean

```diff
@@ -2354,6 +2354,81 @@ theorem bmod_neg_bmod : bmod (-(bmod x n)) n = bmod (-x) n := by
   apply (bmod_add_cancel_right x).mp
   rw [Int.add_left_neg, ← add_bmod_bmod, Int.add_left_neg]
 
+/- ### udiv, ediv -/
+
```

Comment:
What is `udiv`? Can you double-check that Int has an `udiv` operation. I feel what you are referring to is `\` which is called `ediv`. Please double-check this and do a pass over this PR based on what you find out.

Reply:
done, it is indeed `ediv` the one I had in mind